### PR TITLE
Zookeeper

### PR DESCRIPTION
**Pull request by Cheackraze**
## About the PR 
Now introducing the station exclusive role of Zookeeper! The Zookeeper is a part of the Services department, though they only have 'service' level access in order to unlock 'zoo' doors which are planned for Kettle. It is purely an RP role similar to the Reporter and Psychologist.  For now this is just all of the required items to have the role functioning, including sprites, spawner, loadout, and is enabled as a late join for Kettle. After this is merged, Kettle will be set up to house the Zookeeper and their critter companions.

Also adds in a "Service" access restricted windoor prototype because there wasn't one with that access yet, and I feel it would be nice to have a thindoor for those generic service workers anyway

Many thanks to EmoGarbage for the sprites
**Screenshots**

zookeeper 5

**Changelog**

:cl: checkraze
- add: Added the Zookeeper. Find them patrolling and caring for Kettle station's various wildlife soon!

### CODE REVIEW

**Comment by Cheackraze** [comment]
looks like i set it up too early and its failing because I haven't actually added the spawnpoint to kettle. I'm still awaiting [#12011] to be merged before I make more updates to the map

**Review by moonheart08 — APPROVED** [review]

File: Resources/Textures/Clothing/Head/Hats/safarihat.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "discord: checkraze#7501 github: cheakraze",
```
Comment (EmoGarbage404) [inline Resources/Textures/Clothing/Head/Hats/safarihat.rsi/meta.json:4]: please properly credit me. "Created by EmoGarbage404" is fine with CC-0.

File: Resources/Textures/Objects/Devices/pda.rsi/meta.json [hunk 2]
```diff
@@ -164,6 +164,9 @@
       "name": "pda-interntech"
     },
     {
+      "name": "pda-zookeeper"
+    },
+	{
```
Comment (EmoGarbage404) [inline Resources/Textures/Objects/Devices/pda.rsi/meta.json:169]:
```suggestion
    {
```

File: Resources/Textures/Objects/Misc/id_cards.rsi/meta.json [hunk 3]
```diff
@@ -147,6 +147,9 @@
     },
     {
       "name": "idmusician"
+    },
+	{
```
Comment (EmoGarbage404) [inline Resources/Textures/Objects/Misc/id_cards.rsi/meta.json:151]:
```suggestion
    {
```

File: Resources/Prototypes/Entities/Structures/Doors/Windoors/windoor.yml [hunk 4]
```diff
@@ -12,6 +12,14 @@
 
 # TODO remove these with parameterized prototypes/whatever we end up doing
 # Windoors (alphabetical)
+- type: entity
+  parent: Windoor
+  id: WindoorServiceLocked
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Structures/Doors/Windoors/windoor.yml:17]: it says alphabetical and yet you have not placed it in the proper spot.

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
also, not gonna hound you over this, but technically the zookeeper job marker is using the old sprites for the outfit.

File: Resources/Textures/Clothing/Head/Hats/safarihat.rsi/meta.json (hunk 1 above)
Comment (Cheackraze) [reply Resources/Textures/Clothing/Head/Hats/safarihat.rsi/meta.json:4]: My apologies, I will change them when I get home, I forgot thats in the json

File: Resources/Prototypes/Entities/Structures/Doors/Windoors/windoor.yml (hunk 4 above)
Comment (Cheackraze) [reply Resources/Prototypes/Entities/Structures/Doors/Windoors/windoor.yml]: done

File: Resources/Textures/Clothing/Head/Hats/safarihat.rsi/meta.json (hunk 1 above)
Comment (Cheackraze) [reply Resources/Textures/Clothing/Head/Hats/safarihat.rsi/meta.json]: changed thank you again EmoGarbage

**Pull request merged**